docs/source/installation.rst

```diff
@@ -43,10 +44,10 @@ Mac OS X
 Linux
     Use your package manager to install ``numpy`` and ``scipy``
         Ubuntu: ``sudo apt-get install python-numpy python-scipy``
-    Run ``pip install pint``
+    Run ``pip install pint`` (for system python installs, use ``sudo pip``)
```

Comment:
in #777 @galbramc mentioned "You can also add --user to just install in your home directory to avoid using sudo."

Just checking -- do we definitely prefer the sudo option for all of these?